clouddriver-elasticsearch-aws/src/main/kotlin/com/netflix/spinnaker/config/ClusterScalingPolicyTaggingAgentProperties.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.config
+
+import java.time.Duration
+
+data class ClusterScalingPolicyTaggingAgentProperties(
+  /**
+   * [oldServerGroupBoundaryMs] defines how long a server group must exist
+   * before the agent will consider a missing scaling policy as desired.
+   */
+  var oldServerGroupBoundaryMs: Long = Duration.ofMinutes(30).toMillis()
```

Comment:
Thoughts on whether this is too aggressive of a value? 

Perhaps I'm misunderstanding the comment, 30min seems a touch aggressive to be deemed the new norm if something goes sideways?

Reply:
Hmm, you're probably right. I was operating under the idea that a deploy operation won't (typically) take very long, but you're right it doesn't protect us much if things go sideways. I'll set it to something more like 8 hours.